librad/src/git/url.rs

```diff
@@ -15,6 +15,8 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+#![allow(unused)]
```

Comment:
:thinking: what's up with this?

Reply:
I made this module `pub(crate)`, but wanted to keep the conversion functions for future use.